dll/win32/shell32/iconcache.cpp

```diff
@@ -390,8 +423,10 @@ static INT SIC_LoadIcon (LPCWSTR sSourceFile, INT dwSourceIndex, DWORD dwFlags)
     HICON hiconSmall=0;
```

Comment:
```suggestion
    HICON hiconSmall = NULL;
```
and same suggestion for hiconLarge above.

Reply:
Both fixed.